# Float: The length of the trajectory in time domain
# Unit: s
trajectory_time_length: 6.0 # Trajectory length in seconds
# Float: Curve re-sampling step size in m
# Unit: m
curve_resample_step_size: 1.0 
# Integer: Amount to downsample input lanelet centerline data. Value corresponds to saving each nth point.
downsample_ratio: 8 
# Float: Minimum allowable speed
# Unit: m/s
minimum_speed: 2.2352
# Float: Minimum value for lookahead distance
# Unit: m
minimum_lookahead_distance: 5.0
# Float: Maximum value for lookahead distance
# Unit: m
maximum_lookahead_distance: 25.0
# Float: Minimum speed value for lookahead calculation
# Unit: m/s
minimum_lookahead_speed: 2.8
# Float: Maximum speed value for lookahead calculation
# Unit: m/s
maximum_lookahead_speed: 13.9
# Integer: ratio to calculate lookahead distance from speed
lookahead_ratio: 2.0 
# Integer: Size of the window used in the moving average filter to smooth both the computed curvature and output speeds
moving_average_window_size: 5
# Integer: Number of points to look ahead when calculating the curvature of the lanelet centerline
curvature_calc_lookahead_count: 1 
